api/renter.go

```diff
@@ -29,7 +31,6 @@ type FileInfo struct {
 	UploadProgress float32
 	Nickname       string
 	Filesize       uint64
-	Repairing      bool
```

Comment:
Well, we're not using it now and we never did. We can add it back when we start doing repairing, but that may not happen for a while.